src/main/resources/com/tresys/nitf/xsd/nitf.dfdl.xsd

```diff
@@ -125,9 +131,12 @@ The message root is 'NITF'.
     <xs:restriction base="xs:unsignedLong" />
   </xs:simpleType>
 
-  <xs:simpleType name="dateTime" dfdl:lengthKind="explicit" dfdl:length="14" dfdl:calendarPatternKind="explicit" dfdl:calendarPattern="yyyyMMddHHmmss">
-    <xs:restriction base="xs:dateTime" />
-  </xs:simpleType>
+  <xs:complexType name="dateTime">
+    <xs:choice>
+      <xs:element name="dateTime2.1" type="xs:dateTime" dfdl:lengthKind="explicit" dfdl:length="14" dfdl:calendarPatternKind="explicit" dfdl:calendarPattern="yyyyMMddHHmmss" />
+      <xs:element name="dateTime_str" type="xs:string" dfdl:lengthKind="explicit" dfdl:length="14" />
```

Comment:
I'll look into the choice dispatch.  I think that will work.  Originally I had a separate pattern for the NITF 2.0 stuff as well as a string because of discrepancies in the test data, but if we are just choosing between 2.1 and the string form, the dispatch should work.